### ISSUE TITLE
Markers Randomly Jumping Around (ANDROID BUG ONLY)

### ISSUE THREAD

**Issue by 18clans** [dapriett/nativescript-google-maps-sdk#344]
What I'm trying to achieve:
When I favor a marker on Map 1, Map 2 marker with the same ID should be favorited as well.  This is working fine on iOS.

What is the bug:
**Only happens on Android**
Have both Map 1 and Map 2 open.  Go to either tab and click on a marker to show the "Favor Me" button.  If you spam that button, the markers will jump randomly.  When the markers are randomly moved, if you scroll the map, everything goes back to normal.  If you hide one map and try to spam the favor button, this bug does not happen.

Demo Project: [https://github.com/18clans/google-maps-tab-view](url)

NativeScript: 5.3.2
tns-core-modules: 5.3.2
tns-android: 5.3.1
tns-ios: 5.3.1
nativescript-google-maps-sdk: ^2.7.0

**Comment by 18clans** [comment]
I found something on Stack Overflow but not for NativeScript.

https://stackoverflow.com/questions/20131387/jumping-markers-on-google-maps

**Comment by 18clans** [comment]
Found another interesting article. Looks like this is a Google Maps issue and not the plugin.  The plugin will need to update using the latest Google Maps SDK.

https://issuetracker.google.com/issues/35828690

**Comment by Firetrip** [comment]
@18clans did you ever solve that? we are currently also facing the same issue :(

**Comment by 18clans** [comment]
> @18clans did you ever solve that? we are currently also facing the same issue :(

No, this issue was never resolved.  We stopped using NativeScript due to a lot of issues and performance we faced.  Since we switched to ReactNative, it's a night and day difference IMO.
